### PR TITLE
feat: explore page,transiton in carousel & sidebar

### PR DESCRIPTION
**Pull request by pankaj-prog**
Explore page
BlitheBrains - explore

### CODE REVIEW

**Review by m26unkwn — APPROVED** [review]
Could not find issues
1. love the transition in you sidebar
2. CSS is well written

File: src/pages/Explore/VideoCard.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            <span className="info">Playlist</span>
+          </button>
+        </div>
+      </div>
+      <section className="content">
+        <div className="avatar avatar-sm avatar-round">
+          <img className="responsive-img" src={creatorImg} alt={creator} />
+        </div>
+        <div>
+          <div className="title">{title}</div>
```
Comment (bhtibrewal) [inline src/pages/Explore/VideoCard.jsx]: Suggestion: you could use p tag here

File: src/pages/Explore/VideoCard.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          <section className="content-secondary">
+            <div className="video-data text-muted">
+              <span className="views">{views} views </span>
```
Comment (bhtibrewal) [inline src/pages/Explore/VideoCard.jsx:31]: Suggestion: p tag

File: src/pages/Explore/VideoCard.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        <img className="responsive-img" src={thumbnail} alt={title} />
+
+        <div className="cta-wrapper">
+          <button className="btn">
+            <MdOutlineWatchLater />
+            <span className="info">Watch Later</span>
+          </button>
+          <button className="btn">
+            <MdPlaylistPlay />
+            <span className="info">Playlist</span>
```
Comment (bhtibrewal) [inline src/pages/Explore/VideoCard.jsx:19]: Suggestion: use p tag

**Review by bhtibrewal — APPROVED** [review]
good work clean code, you are good to merge

**Pull request merged**